### ISSUE TITLE
Geen rechten om lijst aan te maken

### ISSUE THREAD

**Issue by ocrasborn** [signfuse/corpusNGT#30]
Nieuwe lijst aanmaken als er nog geen is lukt niet: U heeft geen toegangsrechten voor deze pagina. (Idem voor andere lijsten, met Gebaarders, Collega’s). Ik weet niet als wie ik was ingelogd -- zie issue #31 . Als ik opnieuw inlog als onnocras, lukt het wel.

**Comment by signfuse** [comment]
> Dit komt omdat de gebruiker nog niet geverifieerd is.

Edit: verkeerd gezien van mij. De gebruiker was wel geverifieerd (door mijzelf). Maar de gebruiker had nog geen rol toegekend gekregen (tolk, docent of cursist - dat was ik vergeten te doen).

Maar ik vraag me af of het wel nodig is om 3 verschillende rollen en rechten te voorzien (dus tolk, cursist of docent), of krijgt iedereen dezelfde rechten?

**Comment by ocrasborn** [comment]
De rollen zijn niet nodig voor aparte rechten, maar ze leken me nuttig omdat docenten dan lijsten met studenten kunnen aanmaken, en kunnen filteren op student zonder lastig gevallen te worden door andere gebruikers. En voor mij als beheerder om beter te kunnen plaatsen wie er gebruik van maakt bv.

**Comment by ocrasborn** [comment]
@signfuse Bij nader inzien: als we alleen docenten lijsten met gebruikers laten samenstellen (zie #38 ), zou dat een recht zijn dat aan een groep gekoppeld is. Maar als je dat anders wilt oplossen, ook prima hoor.

**Comment by signfuse** [comment]
Ik ben nog niet helemaal mee wat nu de bedoeling is van deze gebruikerslijsten?
Om de andere lijsten (bv zoeklijst) te kunnen delen met iedereen in een gebruikerslijst?

**Comment by ocrasborn** [comment]
Precies, dat! Dus de use case: docenten zoeken in de 2000+ video's een paar interessante, of twee om voor huiswerk te bekijken, zetten die in een lijst met video's ("Huiswerk week 3"). Die lijst willen ze aan hun klas communiceren. De studenten in de klas hebben ze eerder al in een gebruikerslijst gezet ("Klas 2a"), en de videolijst "Huiswerk week 3" wordt dan gedeeld met die groep "Klas 2a". Die zien de lijst dan bij hun eigen videolijsten verschijnen, met op de een of andere manier gevisualiseerd dat de lijst niet door henzelf gemaakt is, maar door iemand anders met ze is gedeeld.

**Comment by signfuse** [comment]
Bijkomende vraag, is die lijst "read-only" voor de anderen? Of mogen ze de lijst ook bewerken?

En, als lijsten gedeeld worden zullen ze publiek worden, ik hoop dat dat okee is.

**Comment by ocrasborn** [comment]
* Anderen mogen de lijst niet bewerken.
* Wat betekent publiek in dit geval? Alleen docenten kunnen gebruikerslijsten zien, toch? Maar dan zien ze elkaars lijsten? Dat lijkt me in orde, zolang het binnen dezelfde instelling is. Maar niet buiten die instelling, dat zouden we af moeten schermen uit privacyoverwegingen.

**Comment by signfuse** [comment]
@ocrasborn ik heb het over de lijst die gedeeld wordt (bv een gedeelde videolijst), niet over de gebruikerslijst. Die lijst gaat dan uit de privacymodus en wordt publiek zodat de anderen hem (bv de gedeelde videolijst) ook kunnen zien. Dus dan kan iedereen hem zien als ze de identifier kennen.
